filament/backend/src/ostream.cpp

```diff
@@ -437,6 +438,11 @@ io::ostream& operator<<(io::ostream& out, BufferDescriptor const& b) {
     << ", user=" << b.getUser() << " }";
 }
 
+utils::io::ostream& operator<<(utils::io::ostream& out,
```

Comment:
nit: can just write `io` instead of `utils::io` 
Same with just `BufferObjectStreamDescriptor` instead of `filament::backend::BufferObjectStreamDescriptor`